Sources/MastodonKit/Client.swift

```diff
@@ -60,4 +60,24 @@ public struct Client: ClientType {
 
         task.resume()
     }
+
+    /// OAuth authorization URL. Use this to initiate a standard OAuth login.
+    ///
+    /// - Parameters:
+    ///   - clientID: The client ID.
+    ///   - scopes: The access scopes.
+    ///   - redirectURI: The client redirectURI.
+    /// - Returns: URL for the OAuth authorization endpoint.
+    public func oauthAuthorizationURL(clientID: String, scopes: [AccessScope], redirectURI: String) -> URL? {
```

Comment:
Wouldn't this be equivalent to `Login.oauth(...)` as implemented [here](https://github.com/MastodonKit/MastodonKit/blob/master/Sources/MastodonKit/Requests/Login.swift#L49).

Reply:
Similar, sure. This PR was from four years ago - I've no idea what or why I was trying to change here anymore.

Reply:
Yup, I'm sorry about the delay. I've recently taken over this repository, so I'm trying to catch up with everything. If you agree, I'll go ahead and close this PR?

Thanks a lot for your contribution nonetheless!

Reply:
> If you agree, I'll go ahead and close this PR?

Yeah, that sounds good to me. Good luck with getting the project up and running again!